Library/DasharoSystemFeaturesUiLib/DasharoSystemFeatures.c

```diff
@@ -211,6 +212,30 @@ DasharoSystemFeaturesUiLibConstructor (
     return Status;
   }
 
+  BufferSize = sizeof (mDasharoSystemFeaturesPrivate.DasharoFeaturesData.MeMode);
+  Status = gRT->GetVariable (
+      mMeModeEfiVar,
+      &mDasharoSystemFeaturesGuid,

```

Comment:
```suggestion
      &gDasharoSystemFeaturesGuid,
```

And every other occurrence.

Reply:
Done